web/main/models.py

```diff
@@ -1292,14 +1312,18 @@ def has_bad_footnotes(self):
         return False
 
 
-class ContentAnnotationQueryset(models.QuerySet):
+class ContentAnnotationQuerySet(models.QuerySet):
     def valid(self):
         """
         Return annotations excluding those that were marked invalid when shifting.
         """
         return self.exclude(global_start_offset=-1, global_end_offset=-1)
 
 
+# (2022-07-19) django type-stubs workaround
```

Comment:
Interesting. Do you know of any documentation/github issue/etc. explaining why this workaround is necessary?

Reply:
Oops, forgot to add: https://github.com/typeddjango/django-stubs#my-queryset-methods-are-returning-any-rather-than-my-model

Will add to the code